samples/bluetooth/public_broadcast_sink/src/bap_broadcast_sink.c

```diff
@@ -0,0 +1,412 @@
+/*
```

Comment:
Isn't it a copy of existing application? Can you extend the existing one with PBP stuff and just guard it with `CONFIG_BT_PBP` ?